src/Sepes.Infrastructure/Service/Study/StudyWbsValidationService.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Sepes.Common.Exceptions;
+using Sepes.Infrastructure.Model;
+using Sepes.Infrastructure.Service.Interface;
+
+namespace Sepes.Infrastructure.Service
+{
+    public class StudyWbsValidationService : IStudyWbsValidationService
+    {
+        readonly ILogger _logger;
+        readonly IWbsValidationService _wbsValidationService;
+
+        public StudyWbsValidationService(ILogger<StudyWbsValidationService> logger, IWbsValidationService wbsValidationService)
+        {
+            _logger = logger;
+            _wbsValidationService =
+                wbsValidationService ?? throw new ArgumentNullException(nameof(wbsValidationService));
+        }
+
+        public async Task ValidateForStudyCreateOrUpdate(Study study)
+        {
+            try
+            {
+                await PerformValidation(study, false);
+            }
+            catch (Exception ex)
+            {
```

Comment:
Code scanning does not like generic catch clauses, should we disable this warning?

Reply:
I agree